official/resnet/keras/keras_cifar_main.py

```diff
@@ -177,7 +177,10 @@ def run(flags_obj):
 
   validation_data = eval_input_dataset
   if flags_obj.skip_eval:
-    tf.keras.backend.set_learning_phase(1)
+    if strategy:
+      # TODO(haoyuzhang): Understand slowdown of setting learning phase when
```

Comment:
Please add a bug number, I know we are committed to resolving this but this makes the test somewhat invalid as OTB.  Should we set this up as a tweak rather than avoid it for `if strategy`. Maybe make a flag for `turn off learning phase` or something.  More work but keeps this honest.